server/sonar-web/src/main/js/apps/projects/components/AllProjects.js

```diff
@@ -72,6 +75,12 @@ export default class AllProjects extends React.PureComponent {
 
   isFiltered = () => Object.keys(this.state.query).some(key => this.state.query[key] != null);
 
+  getSavedOptions = () => ({
+    sort: utils.getSort(),
+    view: utils.getView(),
+    visualization: utils.getVisualization()
```

Comment:
I'm not sure if this logic should be done here where we return the saved options, or where we save them or where retrieve them from the localstorage.